pkg/fileutils/fileutils_test.go

```diff
@@ -384,73 +376,82 @@ func TestMatches(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		res, _ := regexpMatch(test.pattern, test.text)
+		pm, err := NewPatternMatcher([]string{test.pattern})
+		if err != nil {
+			t.Fatalf("invalid pattern %s", test.pattern)
```

Comment:
oh never mind - I thought this was real code, not test code. Up to you if you want to tweak it.